src/index.js

```diff
@@ -1,6 +1,32 @@
-/**
-* Put your JavaScript here
-*/
+let state = {
+  todos: [
+    { title: 'First todo', complete: false },
+    { title: 'Second todo', complete: true },
+    { title: 'Third todo', complete: true },
+  ],
+};
+
+document.onload = init();
+
+function init() {
+	const listNode = document.getElementById('list');
+
+	const filterButton = document.getElementById('filter-button')
+	filterButton.addEventListener('click', (event) => {
```

Comment:
Not sure what you mean here. When you say have the button "say" something, are you referring to giving visual feedback that we are seeing a filtered list, or that we should track the state of "filtered/non-filtered" in state?